.github/workflows/common_hive_reports.yaml

```diff
@@ -63,9 +63,9 @@ jobs:
       - name: Pull image
         if: ${{ inputs.job_type == 'main' }}
         run: |
-          docker pull ghcr.io/lambdaclass/ethrex:unstable
-          # Tests use ethrex:latest so we retag it
-          docker tag ghcr.io/lambdaclass/ethrex:unstable ethrex:ci
+          docker pull ghcr.io/lambdaclass/ethrex:main
+          # Tests use ethrex:ci so we retag it
+          docker tag ghcr.io/lambdaclass/ethrex:main ethrex:ci
```

Comment:
Do tests use `ethrex:ci`? I think hive tests are using `unstable` (now `main`) tag

Reply:
So, Hive uses what the client file specifies https://github.com/lambdaclass/ethrex/blob/main/.github/config/hive/clients.yaml#L4 (assuming you pass it as arg, we do). If no client file is specified, it uses the default: https://github.com/ethereum/hive/pull/1325/files#diff-88221494e3fbc0839756b9c5b02ad47942b9bbcf3970e00c335e8306c0c5009cR2

Reply:
Ah, I forgot that change